helm_deploy/values-dev.yaml

```diff
@@ -13,19 +13,11 @@ db:
 
 ingress:
   enabled: true
-  enable_whitelist: true
+  enable_whitelist: false
   hosts:
-    - host: calculate-journey-variable-payments-dev.hmpps.service.justice.gov.uk
-      cert_secret: calculate-journey-variable-payments-cert
+    - host: calculate-journey-variable-payments-dev.apps.live-1.cloud-platform.service.justice.gov.uk
```

Comment:
Wow that is a bit of a mouthful.  Is that right?

Reply:
Yes, this one of the ways of doing it for non prod environments.  We can have a custom one, but is that really required for dev?